sdcm/nemesis.py

```diff
@@ -381,14 +362,11 @@ def publish_event(self, disrupt, status=True, data=None):
         DisruptionEvent(nemesis_name=disrupt, severity=severity, **data).publish()
 
     def set_current_running_nemesis(self, node):
-        with NEMESIS_TARGET_SELECTION_LOCK:
-            node.running_nemesis = self.current_disruption
+        set_running_nemesis(node, self.current_disruption, self.disruptive)
```

Comment:
how does self.disruptive would work ?

all those are set on only on nemesis classes, not the Nemesis class.